adb-connect, adb-pair, adb-disconnect: add page (#15250)
tldr-pages/tldr: pages/common/adb-disconnect.md

# adb disconnect

> > This command is an alias of `adb connect`.
> 
> - View documentation for the original command:

Author: ```suggestion
> This command has been moved.

- View documentation for `adb disconnect`:
```
Something like this?

Reviewer: This would be okay in my opinion. However, I'd like to wait for other reviewers' opinions on this one.

Author: Can you make a suggestion for what you mean with the second line?
